pkg/config/config.go

```diff
@@ -183,6 +183,11 @@ type LintersSettings struct {
 	Gocyclo struct {
 		MinComplexity int `mapstructure:"min-complexity"`
 	}
+	Cyclop struct {
```

Comment:
@bkielbasa Am I right that this config is never used?

Reply:
@maratori https://github.com/golangci/golangci-lint/pull/1743

Reply:
👌